app/import_functions.go

```diff
@@ -491,10 +491,10 @@ func (a *App) ImportUser(data *UserImportData, dryRun bool) *model.AppError {
 	if data.ProfileImage != nil {
 		file, err := os.Open(*data.ProfileImage)
 		if err != nil {
-			mlog.Error(fmt.Sprint("api.import.import_user.profile_image.error FIXME: NOT FOUND IN TRANSLATIONS FILE", err))
+			mlog.Error(fmt.Sprint("Unable to open the profile image.", err))
```

Comment:
Updated these to `Unable to ... ` - let me know if any concerns.

Reply:
sounds good